plot_orderbook_tokenpair.py

```diff
@@ -153,13 +159,20 @@ def corner(f):
         action='store_true',
         help="Show the output picture in the browser.")
 
+    parser.add_argument(
+        '--network',
+        dest='network',
+        type=str,
+        help="Choose one network (mainnet or rinkeby)")
+
     parser.add_argument(
         '--no-show',
         dest='show',
         action='store_false',
         help="Do not show the output picture in the browser.")
 
     parser.set_defaults(show=True)
+    parser.set_defaults(network='mainnet')
```

Comment:
Same as above - for `no-show` you need the dest here, but you can add the defaults directly to the `add_argument()` calls.